robot/Crisis-Management/resources/ContactPageObject.py

```diff
@@ -0,0 +1,22 @@
+from cumulusci.robotframework.pageobjects import DetailPage
+from cumulusci.robotframework.pageobjects import pageobject
+from BaseObjects import BaseCMPage
+from locators_48 import cm_lex_locators
+
+
+
+@pageobject("Detail", "Contact")
+class ContactDetailPage(BaseCMPage,DetailPage):
+    object_name = "Contact"
+
+    def login_to_community_as_user(self):
+        """ Click on 'Show more actions' drop down and select the option to log in to community as user """
+        locator_actions = cm_lex_locators["contact_locators"]["show_more_actions"]
+        locator_login_link = cm_lex_locators["contact_locators"]["login_to_community"]
+        locator_login_error = cm_lex_locators["contact_locators"]["community_login_error"]
```

Comment:
u r not using locators in 14 and 16 so what is the use of these here?

Reply:
@skristem I actually removed these in the branch you're so when we merge that branch and merge down these should be gone

Reply:
@skristem I forgot to remove it